lib/datapacksutils.js

```diff
@@ -2201,16 +2201,26 @@ DataPacksUtils.prototype.createBulkJob = async function (objectName, operation,
                     VlocityUtils.error('Error', batchInfo);
                 })
                 .on("queue", function (batchInfo) {
+                    VlocityUtils.log(operation + ' batch queued')
+                    //VlocityUtils.log('queue, batchInfo:', batchInfo);
                     batch.poll(1000 /* interval(ms) */, 40000 /* timeout(ms) */);
                 })
                 .on("response", function (rets) {
                     for (var i = 0; i < rets.length; i++) {
-                        if (!rets[i].success) {
-                            VlocityUtils.error('Error', 'Batch Insert Failed', `#${i+1}`, rets[i].errors.join(', '));
+                        if (rets[i].success) {
+                            VlocityUtils.log("#" + (i + 1) + " " + operation + " successful, id = " + rets[i].id);
+                            if (self.vlocity.insertIndexToSfIdMap[randomNum + '-' + i]) {
+                                self.vlocity.insertIndexToSfIdMap[randomNum + '-' + i].id = rets[i].id;
+                                self.vlocity.nameToSfIdMap[self.vlocity.insertIndexToSfIdMap[randomNum + '-' + i].name] = rets[i].id;
+                            }
+                        }
+                        else {
+                            VlocityUtils.log("# " + (i + 1) + " error occurred, message = " + rets[i].errors.join(', '));
```

Comment:
Change to more like above where it uses error instead of log. 

`VlocityUtils.error('Error', 'Batch Insert Failed', `#${i+1}`, rets[i].errors.join(', '));`